Files/Strings/en-US/Resources.resw

```diff
@@ -2622,4 +2622,13 @@
   <data name="NavigationToolbarCopy.Label" xml:space="preserve">
     <value>Copy</value>
   </data>
+  <data name="BundleOptionsFlyoutAddFileMenuItem" xml:space="preserve">
```

Comment:
Shouldn't this be "BundleOptionsFlyoutAddFileMenuItem.Text"?

Reply:
Thanks! Fixed in https://github.com/files-community/Files/pull/5475/commits/88ff9c11d4a61063733cbbc1ca5f47b9c2d1454c